test/lint.spec.ts

```
@@ -0,0 +1,1659 @@
+			mockedExistsSync.mockReturnValue(true);
+			mockedReadFileSync.mockReturnValue('---\nrunner: "yarn dlx"\n---\n');
+
+			expect(readSettings()).toMatchObject({ runner: "yarn dlx" });
+		});
+	});
+
+	describe("readSettings maxLintAttempts", () => {
+		it("should parse maxLintAttempts from frontmatter", () => {
+			expect.assertions(1);
+
+			mockedExistsSync.mockReturnValue(true);
```

ESLint

[error] Unsafe call of a type that could not be resolved.

(ts/no-unsafe-call)